dspace-spring-rest/src/main/java/org/dspace/app/rest/model/DSpaceObjectRest.java

```diff
@@ -20,13 +20,16 @@
  *
  */
 public abstract class DSpaceObjectRest extends BaseObjectRest<String> {
+	public static final String PRJ_DEFAULT = "";
+	public static final String PRJ_CONTEXT = "context";
```

Comment:
I agree with @terrywbrady this should be tracked on a separate ticket.
I'm also not fully convinced that it is a good idea to introduce a so high level of configuration. Projection will have a crucial role in the implementation of the REST client, allowing a DSpace repository to customize such aspect could lead to incompatibility between a client implementation and a generic DSpace repository It will be probably better to allow only addition of new projections and slightly configuration of specific projections (for instance which metadata need to go in a summary). Another point here is that projections need to be fully understood by the Repository classes implementation so to provide the best performance. Implement such optimizations allowing also full configurable projections could be overcomplex